src/components/Assets/AssetsList.tsx

```diff
@@ -68,32 +66,31 @@ const AssetsList = () => {
       qParams.warranty_amc_end_of_validity_after || "",
   };
 
-  const { refetch: assetsFetch, loading } = useTanStackQueryInstead(
-    routes.listAssets,
-    {
-      query: params,
-      onResponse: ({ res, data }) => {
-        if (res?.status === 200 && data) {
-          setAssets(data.results);
-          setTotalCount(data.count);
-        }
-      },
-    },
-  );
+  const {
+    refetch: assetsFetch,
+    isLoading: loading,
```

Comment:
let's not unnecessarily alias things

```suggestion
    isLoading,
```